packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -32,42 +32,44 @@ export function comboboxTemplate<T extends FASTCombobox>(
                         aria-disabled="${x => x.ariaDisabled}"
                         aria-expanded="${x => x.ariaExpanded}"
                         aria-haspopup="listbox"
-                        class="selected-value"
-                        part="selected-value"
+                        class="field"
+                        part="field"
                         placeholder="${x => x.placeholder}"
                         role="combobox"
                         type="text"
                         ?disabled="${x => x.disabled}"
                         :value="${x => x.value}"
                         @input="${(x, c) => x.inputHandler(c.event as InputEvent)}"
                         @keyup="${(x, c) => x.keyupHandler(c.event as KeyboardEvent)}"
-                        ${ref("control")}
+                        ${ref("field")}
                     />
-                    <div class="indicator" part="indicator" aria-hidden="true">
-                        <slot name="indicator">
-                            ${staticallyCompose(options.indicator)}
-                        </slot>
-                    </div>
                 </slot>
                 ${endSlotTemplate(options)}
+                <div class="open-close-icon" part="open-close-icon" aria-hidden="true">
```

Comment:
this is a nit but I'd suggest a span here as the element is intended to be inline.

Reply:
@bheston I need to pull down and test a couple things here on the a11y end of things.